common/modifiers/hmo_hardcoded.txt

```diff
@@ -200,23 +200,23 @@ nation_rank_5 = {
 	monthly_wage_modifier = 0.10
 	max_mercenary_stacks = 4
 
-	agressive_expansion_impact = 0.2
-	agressive_expansion_monthly_change = 0.05
+	agressive_expansion_impact = 0.3
+	agressive_expansion_monthly_change = 0.075
 }
 
 is_tribe = {
 	max_loyalty = 100
 	enslavement_efficiency = 0.025
 	change_governor_policy_cost_modifier = 0.1
-	army_maintenance_cost = 0.25
+	army_maintenance_cost = 0.3
 	global_tribesmen_desired_pop_ratio = 0.1
+	hill_fort_cost = -0.25
 	price_found_city_cost_modifier = 0.25
 	price_revoke_city_status_cost_modifier = -0.2
     price_revoke_metropolis_status_cost_modifier = -0.2
-	monthly_military_experience = 0.30
```

Comment:
this seemed a bit OP, especially when stacked with negative centralization. now negative centralization gives +0.25 which is 0.05 more than civilized govs with their best laws